src/platform/graphics/render-pass.js

```diff
@@ -93,7 +93,7 @@ class RenderPass {
     /** @type {string} */
     name;
 
-    /** @type {import('../graphics/render-target.js').RenderTarget} */
+    /** @type {import('../graphics/render-target.js').RenderTarget|null|undefined} */
```

Comment:
feel free to document these types here for clarity, as it's not obvious:
- `undefined`: render pass does not render to any render target
- `null`: render pass renders to the backbuffer
- `otherwise`: renders to the provided RT

Reply:
Nice, added some markdown to it:

![image](https://github.com/playcanvas/engine/assets/5236548/f4aa71c4-b2d5-4ef6-b72b-05bccd772ccb)